openmetadata-service/src/main/java/org/openmetadata/service/events/ChangeEventHandler.java

```diff
@@ -243,6 +245,13 @@ private List<Thread> getThreads(ContainerResponseContext responseContext, String
     }
 
     if (entityInterface.getChangeDescription() == null) {
+      if (entity instanceof GlossaryTerm || entity instanceof Glossary) {
```

Comment:
@parthp2107 are we not getting any change event for this with ENTITY_CREATED ??

Reply:
Tried:
```
String changeType = responseContext.getHeaderString(RestUtil.CHANGE_CUSTOM_HEADER);
if (RestUtil.ENTITY_CREATED.equals(changeType))
```
Getting changeType as null.

Reply:
<img width="835" alt="Screenshot 2023-05-08 at 6 14 01 PM" src="https://user-images.githubusercontent.com/83201188/236827273-3d481958-5aa5-4ffe-8c53-c71fb659a5ba.png">

Reply:
@parthp2107 lets check why we are not getting entity created change events thats the root issue

Reply:
Addressed in the latest commit.